src/main/java/sorald/cli/Cli.java

```diff
@@ -238,7 +290,7 @@ private RuleViolation parseRuleViolation(String violationSpecifier) {
             int startCol = Integer.parseInt(parts[3]);
             int endLine = Integer.parseInt(parts[4]);
             int endCol = Integer.parseInt(parts[5]);
-            return new SpecifiedViolation(key, fileName, startLine, endLine, startCol, endCol);
+            return new SpecifiedViolation(key, fileName, startLine, startCol, endLine, endCol);
```

Comment:
Targeted repair was apparently entirely broken. The stats collector tests failed miserably before I found this error.